template/web/app/paymaster-bundler/_components/PaymasterBundlerApp.tsx

```diff
@@ -0,0 +1,30 @@
+import { useCallback } from 'react';
+import { PrivyProvider, User } from '@privy-io/react-auth';
+import { privyID } from '../constants';
+import PaymasterBundlerDemo from './PaymasterBundlerDemo';
+
+export default function PaymasterBundlerApp() {
+  const handleLogin = useCallback((user: User) => {
+    console.log(`Successfully authenticated with Privy! Your User ID: ${user.id}`);
```

Comment:
Do we need to log this?  Will the user need their ID?

Reply:
Good point, removed.